File: Java/Estrutura sequencial/soma.java
// Importando biblioteca
import java.util.Scanner;

public class soma {

	public static void main(String[] args) {
		
		// Variáveis
		int x, y, soma;

		Scanner sc = new Scanner(System.in); // Para facilitar a leitura de dados
		
		// Entrada de dados
		System.out.print("Digite o valor de x: ");
		x = sc.nextInt();
		System.out.print("Digite o valor de y: ");
		y = sc.nextInt();
		
		// Algoritmo matemático
		soma = x + y;
		
		// Saída de dados
		System.out.print("A soma de " + x + " + " + y + " é igual a " + soma + ".");
		
		sc.close(); // Boa prática de programação
	}
}

File: C++/Estrutura sequencial/soma.cpp
// Fazer um programa para ler dois valores inteiros X e Y, e depois mostrar na tela o valor da soma destes números.

// A linguagem C++ não lida muito bem com acentos! Então evite usá-los!

// Biblioteca
#include <bits/stdc++.h>

using namespace std;

int main()
{
    // Variáveis
    int x, y, soma;

    // Entrada de dados
    cout << "Digite o valor de X: ";
    cin >> x;

    cout << "Digite o valor de Y: ";
    cin >> y;

    // Algoritmo matemático
    soma = x + y;

    // Saída de dados
    cout << "A soma de " << x << " + " << y << " e igual a " << soma;

    return 0;
}